### ISSUE TITLE
Invoking export-blocks leads to "essential task failed" error messages

### ISSUE THREAD

**Issue by pepyakin** [paritytech/substrate#10583]
When invoking `export-block` command there are error messages about "essential task failed". However, it does not seem to be an issue for exporting blocks since they do get exported. So it's just bad messaging.

```shell
./private/builds/7d8f00b90c/polkadot export-blocks -d /tmp/polkadot-7d8f00b90c-db private/blocks-386880
2022-01-04 16:58:22 Essential task `basic-block-import-worker` failed. Shutting down service.
2022-01-04 16:58:22 Essential task `transaction-pool-task-0` failed. Shutting down service.
2022-01-04 16:58:22 DB path: /tmp/polkadot-7d8f00b90c-db/chains/polkadot/db/full
2022-01-04 16:58:22 Essential task `txpool-background` failed. Shutting down service.
2022-01-04 16:58:22 Essential task `transaction-pool-task-1` failed. Shutting down service.
```

**Comment by bkchr** [comment]
Yeah I have seen similar things. This seems to be correlated with starting only half of the node.

**Comment by dnjscksdn98** [comment]
Faced to the same issue when exporting blocks. Is there any updates on this issue?
But I do also think there's no problems to the exported block data. Just for the error message.

**Comment by ggwpez** [comment]
This also happens when benchmarking sets up a client.  
It does not need the additional components, so shutting them down is fine.  
Maybe the code could be changed to actually never start these components, but `new_partial` does it.

**Comment by abh-swar** [comment]
Hello, was someone able to resolve this issue?

**Comment by nazar-pc** [comment]
I think a few things just need to be kept in memory rather than being dropped, but I didn't bother to dig into it yet as it is harmless.

**Comment by bkchr** [comment]
Yes, as I said above this is because we drop some things that we start as essential task. This will then lead to this log message. And yes, these messages are not problematic.

**Comment by viveksharmapoudel** [comment]
@bkchr  In our case, there is failed error message and no block is being exported....

root@localhost:~# sudo ./polkadot export-blocks --chain kusama --from 20 --to 4000 --base-path /chains_data 
 backup.kusama

2022-12-27 10:31:54 👶 Creating empty BABE epoch changes on what appears to be first startup.
2022-12-27 10:31:54 DB path: /chains_data/chains/ksmcc3/db/full
2022-12-27 10:31:54 Essential task `txpool-background` failed. Shutting down service.
2022-12-27 10:31:54 Exporting blocks from #20 to #4000

root@localhost:~# ls -l
-rw-r--r-- 1 root root         0 Dec 27 10:31 backup.kusama
